### PR TITLE
[chore] Prepare release v1.20.0/v0.114.0

### PR DESCRIPTION
**Pull request by opentelemetrybot**
The following commands were run to prepare this release:
- make chlog-update VERSION=v1.20.0/v0.114.0
- make prepare-release PREVIOUS_VERSION=1[.]19[.]0 RELEASE_CANDIDATE=1.20.0 MODSET=stable
- make prepare-release PREVIOUS_VERSION=0[.]113[.]0 RELEASE_CANDIDATE=0.114.0 MODSET=beta

### CODE REVIEW

**Comment by TylerHelmuth** [comment]
I believe the failing CI is blocked by this release blocker: https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/36393

**Comment by TylerHelmuth** [comment]
https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/36393 is merged and CI now passes.

**Review by TylerHelmuth — APPROVED** [review]

**Review by songy23 — APPROVED** [review]

File: cmd/builder/internal/builder/config.go [hunk 1]
```diff
@@ -66,7 +66,7 @@ type Distribution struct {
 	Name        string `mapstructure:"name"`
 	Go          string `mapstructure:"go"`
 	Description string `mapstructure:"description"`
-	// Deprecated: [v0.113.0] only here to return a detailed error and not failing during unmarshalling.
+	// Deprecated: [v0.114.0] only here to return a detailed error and not failing during unmarshalling.
```
Comment (codeboten) [inline cmd/builder/internal/builder/config.go:69]: should this have stayed at 113?

**Review by codeboten — COMMENTED** [review]
Mostly ok, just one question

**Review by bogdandrutu — APPROVED** [review]

File: cmd/builder/internal/builder/config.go (hunk 1 above)
Comment (bogdandrutu) [inline cmd/builder/internal/builder/config.go:69]:
```suggestion
	// Deprecated: [v0.113.0] only here to return a detailed error and not failing during unmarshalling.
```

Comment (codeboten) [reply cmd/builder/internal/builder/config.go]: submitted a change to revert this in a branch

**Review by dmitryax — APPROVED** [review]

**Pull request merged**